integration_tests/network.py

```
@@ -151,3 +152,82 @@ def connect_custom_mantra():
+            "--miner.etherbase",
+            "0x57f96e6B86CdeFdB3d412547816a82E3E0EbF9D2",
+            "--http.api",
+            "eth,net,web3,debug",
+        ]
+        print(*cmd)
+        proc = subprocess.Popen(
+            cmd,
+            preexec_fn=os.setsid,
+            stdout=logfile,
+            stderr=subprocess.STDOUT,
+        )
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)